sources/packages/web/src/constants/routes/RouteConstants.ts

```diff
@@ -20,6 +20,7 @@ export const InstitutionRoutesConst = {
   LOGIN_WITH_BUSINESS_BCEID: Symbol(),
   INSTITUTION_PROFILE: Symbol(),
   INSTITUTION_PROFILE_EDIT: Symbol(),
+  MANAGE_LOCATION: Symbol(),
```

Comment:
I would recommend changing this to plural "MANAGE_LOCATIONS". This will be the page that will be listing all the institution's locations, right?

Reply:
I will do that